### PR TITLE
Chore: Changelog for v.6.4.0-beta1

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
 # 6.4.0 (unreleased)
```
Comment (dprokop) [inline CHANGELOG.md]:
```suggestion
# 6.4.0-beta1 (2019-09-17)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,57 @@
-# 6.4.0 (unreleased)
+# 6.4.0-beta1 (2019-09-17)
```
Comment (marefr) [inline CHANGELOG.md]:
```suggestion
# 6.4.0 (unreleased)

# 6.4.0-beta1 (2019-09-17)
```

**Review by marefr — APPROVED** [review]
Added a suggestion. LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+* **Alerting**: fix duplicate alert states when the alert fails to save to the database. [#18216](https://github.com/grafana/grafana/pull/18216), [@kylebrandt](https://github.com/kylebrandt)
+* **Alerting**: fix response popover prompt when add notification channels. [#18967](https://github.com/grafana/grafana/pull/18967), [@lzdw](https://github.com/lzdw)
+* **CloudWatch**: Fix alerting for queries with Id (using GetMetricData). [#17899](https://github.com/grafana/grafana/pull/17899), [@alex-berger](https://github.com/alex-berger)
+* **Explore**: Fix auto completion on label values for Loki. [#18988](https://github.com/grafana/grafana/pull/18988), [@aocenas](https://github.com/aocenas)
+* **Explore**: Fixes crash using back button with a zoomed in graph. [#19122](https://github.com/grafana/grafana/pull/19122), [@hugohaggmark](https://github.com/hugohaggmark)
+* **Fix**: Fixes so queries in Explore are only run if Graph/Table is shown. [#19000](https://github.com/grafana/grafana/pull/19000), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (marefr) [inline CHANGELOG.md]: Instead of Fix it should probably be Explore. Nitpick.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,60 @@
...
```
Comment (dprokop) [inline CHANGELOG.md]:
```suggestion
* **Explore**: Fixes so queries in Explore are only run if Graph/Table is shown. [#19000](https://github.com/grafana/grafana/pull/19000), [@hugohaggmark](https://github.com/hugohaggmark)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,60 @@
...
 
+# 6.4.0-beta1 (2019-09-17)
+
+### Features / Enhancements
+* **API**: Readonly datasources should not be created via the API. [#19006](https://github.com/grafana/grafana/pull/19006), [@papagian](https://github.com/papagian)
+* **Alerting**: Include configured AlertRuleTags in Webhooks notifier. [#18233](https://github.com/grafana/grafana/pull/18233), [@dominic-miglar](https://github.com/dominic-miglar)
+* **Annotations**: Add annotations support to Loki. [#18949](https://github.com/grafana/grafana/pull/18949), [@aocenas](https://github.com/aocenas)
+* **Annotations**: Use a single row to represent a region. [#17673](https://github.com/grafana/grafana/pull/17673), [@ryantxu](https://github.com/ryantxu)
+* **Auth**: Allow inviting existing users when login form is disabled. [#19048](https://github.com/grafana/grafana/pull/19048), [@548017](https://github.com/548017)
+* **Azure Monitor and Log Analytics converted and separated into components**. [#18259](https://github.com/grafana/grafana/pull/18259), [@shavonn](https://github.com/shavonn)
```
Comment (dprokop) [inline CHANGELOG.md]:
```suggestion
```

**Review by torkelo — APPROVED** [review]
looks good

**Pull request merged**